api/envoy/annotations/deprecation.proto

```diff
@@ -5,18 +5,24 @@ package envoy.annotations;
 import "google/protobuf/descriptor.proto";
 
 // [#protodoc-title: Deprecation]
-// Allows tagging proto fields as fatal by default. One Envoy release after
-// deprecation, deprecated fields will be disallowed by default, a state which
-// is reversible with :ref:`runtime overrides <config_runtime_deprecation>`.
+// Adds annotations for deprecated fields and enums to allow tagging proto
+// fields as fatal by default and the minor version in which the field was
+// deprecated. One Envoy release after deprecation, deprecated fields will be
+// disallowed by default, a state which is reversible with
+// :ref:`runtime overrides <config_runtime_deprecation>`.
 
 // Magic number in this file derived from top 28bit of SHA256 digest of
-// "envoy.annotation.disallowed_by_default"
+// "envoy.annotation.disallowed_by_default" and "envoy.annotation.minor_ver"
 extend google.protobuf.FieldOptions {
   bool disallowed_by_default = 189503207;
+
+  uint32 minor_ver = 240115895;
```

Comment:
Given that things are already pretty verbose, e.g. `envoy.annotation.minor_ver`, it might be clearer to make it the full `envoy.annotation.deprecated_at_minor_ver`.

Reply:
Done.